packages/@adobe/spectrum-css-temp/components/tags/index.css

```diff
@@ -112,3 +112,11 @@ governing permissions and limitations under the License.
     }
   }
 }
+
+.spectrum-Tags-fieldWrapper.spectrum-Tags-fieldWrapper {
+  width: unset;
+  
+  &:not(.spectrum-Tags-fieldWrapper--positionSide) {
+    display: grid;
```

Comment:
Out of curiosity, what case did this cover? Was Field's `inline-grid` not working?

Reply:
correct, inline-grid won't expand as a block level element would in a parent with no defined width, so if it collapses, it'd never un-collapse